README.md

```diff
@@ -120,7 +122,11 @@ const layerConfigurations = [
   {
     growEditionSizeTo: 5,
     layersOrder: [
-      { name: "Background" },
+      { name: "Background" , {
```

Comment:
@nftchef HI, nice work, one small think. I reckon that this pieces of info in the README file is not set properly. It should something like:

`
{ 
        name: "Background", 
        options: {
             bypassDNA: true
        }
}
`

shouldn't it?

Reply:
yes, you are correct! good catch